src/main/java/mdlaf/components/menu/MaterialMenuUI.java

```diff
@@ -65,7 +68,7 @@ public void uninstallUI(JComponent c) {
 
 	@Override
 	public void paint (Graphics g, JComponent c) {
-		super.paint (g, c);
+		super.paint (MaterialDrawingUtils.getAliasedGraphics (g), c);
```

Comment:
This confuses me because `BasicMenuItemUI#paintMenuItem` will not call `MaterialMenuItemUI#paint` and so not use its anti-aliased Graphics.
This is evidenced with a simple test run:
![no AA in MaterialMenuUI](https://i.vinceh121.me/GoMqFZEw.png)
![with AA in MaterialMenuUI](https://i.vinceh121.me/C4hajtUY.png)

Reply:
Mh! This is a very good catch. Thanks for pointing me to this problem. This night I will make more tests in JDK 8 and after I will accept this PR.